### DISCUSSION TITLE
Use Solution of Steady State as initial condition for MultiApp Transient Solution

### DISCUSSION THREAD

**Discussion by TLWise** [idaholab/moose#21964] [Q&A Modules: General] [answered]
I have a geometry that I have successfully solved the physics for a steady state. I would like to make this solutions the initial conditions for the start of my transient multiapp solution. How do I accomplish this in moose? Is it possible to pull the solution for the initial mesh values of the transient app from the steady state output file?

**Comment by GiudGiud** [comment, marked as answer]
Hello

you may use either the checkpoint or exodus restart systems to achieve this. 
The checkpoint system requires you to keep the exact same variables between both simulations. The exodus one is a little more lenient but you ll have to be careful to restart everything you need
https://mooseframework.inl.gov/application_usage/restart_recover.html

Guillaume

**Reply by TLWise** [reply]
Thank you very much! The information you shared worked.
